UPGRADE-2.3.md

```diff
@@ -0,0 +1,12 @@
+UPGRADE FROM 2.3.3 to 2.3.4
```

Comment:
Ok i understand but the 2.4 version i very difficult to release and i can't use it on production, have you some good news for 2.4 release ?